static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -73,25 +75,26 @@ export function MessageFormatter({breadcrumb}: MessageFormatterProps) {
 
     logMessage = [formattedMessage, ...restArgs].join(' ').trim();
   } else if (
-    breadcrumb.message?.includes('[object Object]') ||
-    Array.isArray(message) ||
-    args.length
+    breadcrumb.data?.arguments.length === 1 &&
+    isObject(message) &&
+    objectIsEmpty(message)
   ) {
+    // There is a special case where `console.error()` is called with an Error object.
+    // The SDK uses the Error's `message` property as the breadcrumb message, but we lose the Error type,
+    // resulting in an empty object in the breadcrumb arguments. In this case, we
+    // only want to use `breadcrumb.message`.
+    logMessage = breadcrumb.message || JSON.stringify(message);
```

Comment:
In what cases do we need the `JSON.stringify()`? Is it covered in a test case?

Reply:
Just in case breadcrumb.message is undefined, then we can print the empty object